src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -16,6 +16,7 @@
 #include "../../cascadia/terminalcore/ITerminalInput.hpp"
 
 #include <til/ticket_lock.h>
+#include <til/winrt.h>

```

Comment:
TerminalCore isn't _really_ a winrt layer, so we should be careful. This is fine, but just food for future thought :)

Reply:
well that's not entirely true, it does have `winrt/Windows.Foundation.h` in it's pch. We are playing games with it though, that's for sure